apps/picsa-tools/resources-tool/src/app/pages/resources/search.module.ts

```diff
@@ -0,0 +1,22 @@
+import { SearchComponent } from "./search.component";
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { FormsModule } from "@angular/forms";
+import { ResourcesMaterialModule } from "../../material.module";
+import { ResourcesComponentsModule } from "../../components/components.module";
+import { PicsaCommonComponentsModule } from '@picsa/components';
+import { PicsaTranslateModule } from '@picsa/shared/modules';
+
+const routes: Routes = [{ path: '', component: SearchComponent, title: 'Search' }];
+
+@NgModule({
+    // declarations:[SearchComponent],
```

Comment:
nit(non-blocking)
If code isn't required it is better to delete rather than comment out. If there is really good reason to keep (e.g. you know for sure it will be required in near-future) then that should be included in a comment - but usually you can still just delete and if needed in the future recover.